packages/runtime/src/runtime/menu.tsx

```diff
@@ -36,27 +46,32 @@ interface MenuItem {
   divider?: boolean;
   hasNotifications?: boolean;
 }
-
-interface MenuBaseProps {
-  items: MenuItem[];
-  styles?: {
+interface MenuStyles {
+  backgroundColor?: TypeColors;
+  labelColor?: TypeColors;
+  selectedColor?: TypeColors;
+  labelFontSize?: number;
+  searchBoxColor?: TypeColors;
+  iconWidth?: string;
+  iconHeight?: string;
+  width?: string;
+  headerCollapsedWidth?: string;
+  footerCollapsedWidth?: string;
```

Comment:
These are probably not needed? In collapsed state the header/footer can use expressions to render a different element